app/views/devise/passwords/new.html.erb

```diff
@@ -1,21 +1,16 @@
-<%= stylesheet_link_tag "application"%>
-
-<div class="logger_in_form">
-
 <h2>Forgot your password?</h2>
 
 <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
   <%= render "devise/shared/error_messages", resource: resource %>
 
-  <div class="field custom-login">
+  <div class="field">
     <%= f.label :email %><br />
     <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
   </div>
 
   <div class="actions">
-  <button class="mybtn" type="button"><%= f.submit "Send me reset password instructions" %><button>
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] @team great work on your app, however when I click the send me reset password instructions am getting the error below 
![image](https://user-images.githubusercontent.com/61408860/208190251-e0f2aea9-5454-4ce6-9d3d-2efdfd7851af.png)